### PR TITLE
Miracles in ms e.39

### PR DESCRIPTION
**Pull request by MassimoVilla**
Here comes the set of miracles included in MS e.39 plus one not included in this MS but published and translated by Paolo Marrassini in his _"Vita", "Omelia", "Miracoli" del santo Gabra Manfas Qeddus_, so that all edited miracles are now included in the BM database.

### CODE REVIEW

File: new/LIT6778MiracleGMQ.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                    <relation name="saws:formsPartOf" active="LIT6778MiracleGMQ" passive="LIT3977Taammer"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethe39"/>.</note>
+                    <ab>
+                        ተአምሪሁ፡ ለአቡነ፡ ገብረ፡ መንፈስ፡ ቅዱስ፡ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስሌነ፡ አሜን።
+                        ዘኮነ፡ በዕለተ፡ ሞቱ፡ ለአቡነ፡ ገብረ፡ መንፈስ፡ ቅዱስ፡ በከዩ፡ ፳ አናብስት፡ ወ፳ አናምርት፡ ወበከዩ፡ አድባር፡ ወአውግር፡ ወበከዩ፡ አዕዋፈ፡ ሰማይ፡ 
+                        ወጸልመ፡ ፀሐይ፡ ወወርኅ፡ ወከዋክት፡ ሰማይ፡ ወምድር፡ ርዕደት፡ ወአድለቅለቁ፡ መቃብራት፡ ወነጐድጓድ፡ በይእቲ፡ ዕለት፡
```
Comment (thea-m) [inline new/LIT6778MiracleGMQ.xml:83]: ከዋክብት?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, that's really wonderful!

File: new/LIT6765MiracleGMQ.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6765MiracleGMQ" passive="LIT3977Taammer"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethe39"/>.</note>
+                    <ab>
+                        በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ንወጥን፡ ዜና፡ ገድሉ፡ ወተአምራቲሁ፡ ለብፁዕ፡ ወለቅዱስ፡
+                        አቡነ፡ ገብረ፡ መንፈስ፡ ቅዱስ፡ ወእምድረዝ፡ ዓርገ፡ ሰማያት፡ ነፍሱ፡ ወሥጋሁ፡
```
Comment (MarcinKrawczuk) [inline new/LIT6765MiracleGMQ.xml:82]: ወእምድኅረዝ፡ ?

File: new/LIT6772MiracleGMQ.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                </listBibl>
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6772MiracleGMQ" passive="LIT3977Taammer"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethe39"/>.</note>
+                    <ab>
+                        ተአምሪሁ፡ ለአቡነ፡ገብረ፡ መንፈስ፡ ቅዱስ፡ ጸሎቱ፡ ወበረከቱ፡
```
Comment (MarcinKrawczuk) [inline new/LIT6772MiracleGMQ.xml:81]: Add space

**Review by MarcinKrawczuk — APPROVED** [review]

File: new/LIT6769MiracleGMQ.xml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethe39"/>.</note>
+                    <ab>
+                        ተአምሪሁ፡ ለአቡነ፡ ገብረ፡ መንፈስ፡ ቅዱስ፡ ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡ ገብሩ፡ 
+                        <del rend="erasure"/> <add place="overstrike">ማርያም፡</add> ለዓለመ፡ ዓለም፡ አሜን።
+                        ወነበረት፡ አሐቲ፡ መበለት፡ እንዘ፡ ትገብር፡ ብዙኃ፡ ኃጢአተ፡ ከመ፡ ሰብአ፡ ዓለም። ወእምዝ፡ ፀንሰት፡ ወአእመራ፡ ሊቀ፡ አበው፡ 
+                        ከመ፡ ፀንሰት፡ ወለከ፡ ኀበ፡ መምህራ። መምህራሰ፡ መዓትም፡ ውእቱ፡ ወሶበ፡ ሰምዓ፡ ዘንተ፡ ነገረ፡ ለአከ፡ እንዘ፡ ይብል፡ በሣልስት፡
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply new/LIT6769MiracleGMQ.xml:84]: you are right, one letter between ወ and ለ is no more legible. 
I corrected the word into `ወ<gap reason="illegible" unit="chars" quantity="1"/>ለከ፡`

**Comment by MassimoVilla** [comment]
All corrections have been incorporated and the explicits added. Thank you!

**Comment by MassimoVilla** [comment]
Merging is still blocked, one more approval is still needed after the suggested changes have been made

**Review by thea-m — APPROVED** [review]
Thank you, sorry for keeping you so long

**Pull request merged**